### PR TITLE
Cards info panel now aligns card number and hidden dots

### PR DESCRIPTION
**Pull request by rluodev**
now fixed like sarthak showed

See https://github.com/hackclub/hcb/issues/5143 for screenshots

### CODE REVIEW

**Review by garyhtou — APPROVED** [review]
yay! so satisfying now.

good to merge after approval from @sarthaktexas

**Review by cjdenio — CHANGES_REQUESTED** [review]
Screenshot 2024-02-05 at 5 47 45 PM

It's still not quite aligned— `items-end` pushes the dots towards the bottom of the container. `items-center` should do the trick!

**Comment by sarthaktexas** [comment]
NO @cjdenio do items-center and then reveal the number… the icon causes it to not be centered even when you do that

**Comment by cjdenio** [comment]
I did, and it was... can you share a screenshot of the text being misaligned with `items-center`, then correctly aligned with `items-end`?

**Comment by sarthaktexas** [comment]
ykkk i can't actually recreate it anymore

File: app/views/stripe_cards/_details.html.erb
Comment (sarthaktexas) [inline app/views/stripe_cards/_details.html.erb]:
```suggestion
            <div class="flex justify-between mono items-center">
```

**Review by sarthaktexas — CHANGES_REQUESTED** [review]

**Review by cjdenio — APPROVED** [review]

**Review by sarthaktexas — APPROVED** [review]

**Pull request merged**
